nixos/modules/services/networking/spacecookie.nix

```diff
@@ -18,19 +23,58 @@ in {
       hostname = mkOption {
         type = types.str;
         default = "localhost";
-        description = "The hostname the service is reachable via. Clients will use this hostname for further requests after loading the initial gopher menu.";
+        description = ''
+          The hostname the service is reachable via. Clients
+          will use this hostname for further requests after
+          loading the initial gopher menu.
+        '';
       };
 
       port = mkOption {
         type = types.port;
         default = 70;
-        description = "Port the gopher service should be exposed on.";
+        description = ''
+          Port the gopher service should be exposed on. The
+          firewall is not opened automatically.
+        '';
       };
 
       root = mkOption {
         type = types.path;
         default = "/srv/gopher";
-        description = "The root directory spacecookie serves via gopher.";
+        description = ''
+          The root directory spacecookie serves via gopher.
+        '';
+      };
+
+      package = mkOption {
```

Comment:
Or you can use overlays. Either way works.

Reply:
I think I want to keep this as an option here if somebody wants it as it is trivial to implement. Overriding a haskellPackage alone is confusing enough…

Reply:
Sounds good 👍